portal/src/main/webapp/js/src/oncoprint/setup.js

```diff
@@ -318,7 +318,7 @@ var comparator_utils = {
     'makeGeneticComparator': function (distinguish_mutation_types, distinguish_recurrent) {
 	var fusion_key = 'disp_fusion';
 	var cna_key = 'disp_cna';
-	var cna_order = makeComparatorMetric(['amp', 'homdel', 'gain', 'hetloss', 'diploid', undefined]);
```

Comment:
This looks similar to SortUtils

https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/lib/SortUtils.ts

Reply:
Just deleted this file (oncoprint directory is obsolete)